.env

```diff
@@ -1 +1,2 @@
+COMPOSE_PROJECT_NAME=pattern-library
```

Comment:
this may wreak havoc for local users as container names clash with the other `pattern-library`, but it's the correct name